packaged/lisk_snapshot_verify.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#
+# LiskHQ/lisk-scripts/lisk_snapshot_verify.sh
+# Copyright (C) 2018 Lisk Foundation
+#
+# Connects source and target versions of Lisk in order to migrate
+# gracefully between protocol changes.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+set -eo pipefail
+DB_NAME="lisk_verify"
+if [ "$USER" == "root" ]; then
+        echo "Error: $0 should not be run be as root. Exiting."
+        exit 1
+fi
+
+if [[ ! $1 ]] ; then
+	echo -e 'Error: snapshot is required'
+	echo -e "Usage: $0 <PATH_TO_SNAPSHOT>\\n"
+	echo -e "Example: $0 backups/lisk_main_backup-8675309.gz"
+	exit 1;
+else
+	DB_SNAPSHOT="$1"
+fi
+
+
+cd "$(cd -P -- "$(dirname -- "$0")" && pwd -P)" || exit 2
+# shellcheck disable=SC1090
+. "$PWD/shared.sh"
+# shellcheck disable=SC1090
+. "$PWD/env.sh"
+
+if [ ! -f "$(pwd)/app.js" ]; then
```

Comment:
If this weren't a lisk installation, wouldn't `shared.sh` and `env.sh` (above) also be missing? Is this check really necessary?